edxval/api.py

```diff
@@ -198,8 +230,7 @@ def get_video_transcript_url(video_id, language_code):
         video_id: it can be an edx_video_id or an external_id extracted from external sources in a video component.
         language_code: language code of a video transcript
     """
-    video_transcript = get_video_transcript(video_id, language_code)
-
+    video_transcript = VideoTranscript.get_or_none(video_id, language_code)
```

Comment:
Don't we have url in serialized data ?

Reply:
I think we should open a cleaning PR at the end to remove such unused api utils all together.